doc-content/enterprise-only/springboot/bus-app-con-app-prop_proc.adoc

```diff
@@ -0,0 +1,131 @@
+
+.Procedure
+. Unzip the `<business-application>.zip` file and navigate to the `<business-application>/<business-application>-service/src/main/resources` folder.
+. Open the `application.properties` file in a text editor.
+. Configure the host, port, and path for the REST endpoints, for example:
++
+[source, bash]
+----
+server.address=localhost
+server.port=8090
+
+cxf.path=/rest
+----
+
+. Configure the {KIE_SERVER} (`kieserver`) so that it can be more easily identified, for example:
++
+[source, bash]
+----
+kieserver.serverId=<business-application>-service
+kieserver.serverName=<business-application>-service
+kieserver.location=http://localhost:8090/rest/server
+kieserver.controllers=http://localhost:8080/jbpm-console/rest/controller
+----
++
+.kieserver parameters
+[cols="30%,15%,55%", options="header"]
+|===
+|Parameter
+|Values
+|Description
+
+|`kieserver.serverId`
+|string
+|The id used to identify the business application when connecting to the {CONTROLLER}.
+
+|`kieserver.serverName`
+|string
+|The   name used to identify the business application when connecting to the {CONTROLLER}. Can be the same string used for the `kieserver.serverId` parameter.
+
+|`kieserver.location`
+|URL
+|Used by other components that use the REST API to identify the location of this server. Do not use the location as defined by `server.address` and `server.port`.
+
+|`kieserver.controllers`
+|URLs
+|A comma-separated list of controller URLs.
+
+|===
+ifdef::PAM[]
+. To enable asynchronous execution, set the value of the `jbpm.executor.enabled` parameter to `true`, uncomment the other `jbpm.executor` parameters, and change the values as required, for example:
++
+[source, bash]
+----
+jbpm.executor.enabled=true
+jbpm.executor.retries=5
+jbpm.executor.interval=0
+jbpm.executor.threadPoolSize=1
+jbpm.executor.timeUnit=SECONDS
+----
+
++
+.Executor parameters
+[cols="30%,15%,55%", options="header"]
+|===
+|Parameter
+|Values
+|Description
+
+|`jbpm.executor.enabled`
+|`true, false`
+|Disables or enables the executor component.
+
+|`jbpm.executor.retries`
+|integer
+|Specifies the number of retries if errors occur while a job is running.
+
+|`jbpm.executor.interval`
+|integer
+|Specifies the length of time that the executor uses to synchronize with the database. The unit of time is specified by the `jbpm.executor.timeUnit` parameter. Disabled by default (value `0`).
+
+|`jbpm.executor.threadPoolSize`
+|integer
+|Specifies the thread pool size.
+
+
+|`jbpm.executor.timeUnit`
+|string
+|Specifies the time unit used to calculate the interval that the executor uses to synchronize with the database. The value must be a valid constant of `java.util.concurrent.TimeUnit`. The default value is `SECONDS`.
+
+|===
+endif::[]
+. If you selected *Business Automation* when you created your business application, specify the components that you want to start at runtime.
```

Comment:
@emmurphy1, right, I'd definitely check that exact question, because the point at which you choose Business Automation or Decision Management or Optimizer in that site is where you're deciding to go with PAM or DM components. Once you select one of those options, there's no more component configuration that occurs, just the package and GAV names, and then you're off. So I think it's an oversight from the SMEs, and am fairly certain that the Business Automation option just doesn't apply to DM, or if used, does the same thing as the Decision Management option.

Either way, it's still confusing in the DM doc for you to leave it as is because earlier in the instructions you say to select the Decision Management option and press on, then later, here, you say "if you selected Business Automation ..." (wait, didn't you tell me to choose Decision Management?). Then the list of components that follow are those that apply to Decision Management option and exclude the additional components available in the Business Automation option, hence your ifdefs that you have in that section of the doc.

Anyway, just sayin'.